dm/pkg/schema/tracker.go

```diff
@@ -99,16 +99,25 @@ type DownstreamTableInfo struct {
 	WhereHandle *sqlmodel.WhereHandle
 }
 
-// NewTracker creates a new tracker. `sessionCfg` will be set as tracker's session variables if specified, or retrieve
+// NewDumpTracker simply returns an empty Tracker,
+// which should be followed by a subsequent initialization.
+func NewDumpTracker() *Tracker {
+	return &Tracker{}
+}
+
+// Init initializes the Tracker. `sessionCfg` will be set as tracker's session variables if specified, or retrieve
 // some variable from downstream using `downstreamConn`.
 // NOTE **sessionCfg is a reference to caller**.
-func NewTracker(
+func (tr *Tracker) Init(
 	ctx context.Context,
 	task string,
 	sessionCfg map[string]string,
 	downstreamConn *dbconn.DBConn,
 	logger log.Logger,
-) (*Tracker, error) {
+) error {
+	if tr == nil {
```

Comment:
when will tr be nil?

if it's for test, we can change test instead

Reply:
This line is copied from the old version. I remember it will let the `load_task` IT fail if deleted (probably this test case tries to `pause-task` before initializing the syncer). will take a look tomorrow.